### PR TITLE
ui: Fixes RTT display, by ensuring use of ember proxies in tomography

### PR DESCRIPTION
**Pull request by johncowen**
Previously the tomography wasn't using ember `get` so proxy updates weren't working, specifically here whilst receiving a blocking update.

This adds `get` usage to the tomography graph component until we update to newer `get`less ember

Skipped tests are adding here to nag us to come back here at some point. Extra brackets seem to have come from prettier.

### CODE REVIEW

File: ui-v2/app/components/tomography-graph.js [hunk 1]
```diff
@@ -28,8 +28,8 @@ export default Component.extend({
     ];
   }),
   distances: computed('tomography', function() {
-    const tomography = this.get('tomography');
-    let distances = tomography.distances || [];
+    const tomography = get(this, 'tomography');
+    let distances = get(tomography, 'distances') || [];
```
[the remark this answers is not in the document]
Comment (johncowen) [reply ui-v2/app/components/tomography-graph.js]: Ah thanks yeah good spot! 🦅 👀 😄

Let me look and see, I think even if it is static we should probably use `get` also

[the remark this answers is not in the document]
Comment (johncowen) [reply ui-v2/app/components/tomography-graph.js]: Hey @meirish I'm pretty good to merge this, but would be good if you could cast your eye over again - I kept thinking to myself 'is it _really_ this' here, so would be good to get a second opinion (you'll see what I mean when you look at it).

TLDR; Looks like `n` was `distances.length` so I stopped passing it through as an extra property and just referenced `distances.length` where it was needed instead.

[the remark this answers is not in the document]
Comment (meirish) [reply ui-v2/app/components/tomography-graph.js]: Oh nice - I could see it being useful if you wanted to tune the sampling rate separately, but it's been like this for a while so it doesn't seem like it'd change, so +1 for just using distances.length.

[the remark this answers is not in the document]
Comment (johncowen) [reply ui-v2/app/components/tomography-graph.js]: Thanks Matthew, this was from the v1 UI and I think it was a contribution from someone also, so good to get a second opinion here!
